packages/admin-panel/src/dataTables/DataTableEditFields.js

```diff
@@ -5,7 +5,13 @@
 
 import React, { useEffect, useMemo, useState } from 'react';
 import styled from 'styled-components';
-import { Select, TextField, DataTable, FetchLoader } from '@tupaia/ui-components';
+import {
+  Select,
+  TextField,
+  DataTable,
+  FetchLoader,
+  Autocomplete as UIAutocomplete,
```

Comment:
Only suggestion would be to rename this component to something a bit clearer, e.g. SqlAutocomplete